packages/documentation/src/pages/forms/form-tutorial-advanced.mdx

```diff
@@ -146,6 +146,13 @@ const formConfig = {
             widgetProps: {
               Y: { 'aria-describedby': 'some_id' },
               N: { 'aria-describedby': 'different_id' },
+            },
+            // Only added to the radio when it is selected
+            // a11y requirement: aria-describedby ID's *must* exist on the page;
+            // and we conditionally add content based on the selection
+            selectedProps: {
+              Y: { 'aria-describedby': 'some_id' },
+              N: { 'aria-describedby': 'different_id' },
             }
```

Comment:
Is this meant to be the same as what's above in `widgetProps`?

Reply:
oops, no

Reply:
The reason this change was made is because an `aria-describedby` containing an ID that doesn't exist will cause an accessibility error in axe.